<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickWindow</class>
 <widget class="QMainWindow" name="QuickWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMake">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>開始製作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeliver">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>切割與上傳</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMsg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>224</y>
      <width>380</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbShowMain">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>240</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCover">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>400</width>
      <height>210</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
